### DISCUSSION TITLE
Is it possible to exclude a dependency from a created PR ?

### DISCUSSION THREAD

**Discussion by alexanderwe** [renovatebot/renovate#28356] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

Hey, 

I was looking through the documentations but I couldn't find a solution to my question. I hope it's fine to ask it here then.

Is it possible when renovate creates an PR, to exclude a certain dependency from being updated ? So for example the bot created a PR which updates Dep A, Dep B and Dep C. I recognise that I do not want to update Dep B but only Dep A and Dep C. Is there a way to tell Renovate to remove Dep B from the PR or is this then a manual step ? I still want to check for updates on Dep B, I just want to exclude it on demand from a given PR. 

Thanks a lot in advance for any help!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Unfortunately it's not supported to "on demand" exclude a dependency from a PR - it needs to be through config

**Reply by alexanderwe** [reply]
@rarkins Alright, thanks a lot for the fast answer 👍
